src/features/tabout.ts

```diff
@@ -3,8 +3,75 @@ import { replaceRange, setCursor, getCharacterAtPos } from "src/utils/editor_uti
 import { Context } from "src/utils/context";
 
 
+const RIGHT_TOKEN = "\\right";
+const CLOSING_BRACKETS = [
+	")",
+	"]", "\\rbrack",
+	"}",
+	"\\}", "\\rbrace",
+	"\\rangle",
+	"\\rceil", "\\rfloor",
+	"\\urcorner",
+	"\\vert",
+	"\\|", "\\Vert",
+	"$"
+];
+const DELIMITERS = [
+	"(", ")",
+	"[", "]", "\\lbrack", "\\rbrack",
+	"\\{", "\\}", "\\lbrace", "\\rbrace",
+	"<", ">", "\\langle", "\\rangle", "\\lt", "\\gt",
+	"\\lfloor", "\\rfloor", "\\lceil", "\\rceil",
+	"/", "\\\\", "\\backslash",
+	"|", "\\vert",
+	"\\|", "\\Vert",
+	"\\uparrow", "\\downarrow", "\\Uparrow", "\\Downarrow",
+	"\\ulcorner", "\\urcorner",
+	"."
+];
```

Comment:
Can we add `>` and `|` to `CLOSING_BRACKETS` to maintain the original behaviour? Additionally, can we align the order of all the delimiters in `CLOSING_BRACKETS` and `DELIMITERS`?

```suggestion
const CLOSING_BRACKETS = [
	")",
	"]", "\\rbrack",
	"}",
	"\\}", "\\rbrace",
	">", "\\rangle",
	"|", "\\vert",
	"\\|", "\\Vert",
	"\\rfloor",
	"\\rceil",
	"\\urcorner",
	"$"
];
const DELIMITERS = [
	"(", ")",
	"[", "]", "\\lbrack", "\\rbrack",
	"\\{", "\\}", "\\lbrace", "\\rbrace",
	"<", ">", "\\langle", "\\rangle", "\\lt", "\\gt",
	"|", "\\vert",
	"\\|", "\\Vert",
	"\\lfloor", "\\rfloor",
	"\\lceil", "\\rceil",
	"\\ulcorner", "\\urcorner",
	"/", "\\\\", "\\backslash",
	"\\uparrow", "\\downarrow", "\\Uparrow", "\\Downarrow",
	"."
];
```

Reply:
You can also refer to the branch I created:
https://github.com/Testudinidae/obsidian-latex-suite/tree/feature/user-input-closing-brackets
https://github.com/Testudinidae/obsidian-latex-suite/pull/1/files

Reply:
Allowing the user to define their own delimiters sounds fine to me. Feel free to merge that commit into this branch!

> The > and | symbols have different meanings in various contexts—they can be used as opening or closing brackets, but they also represent "greater than" and "separator" in other cases. In fact, I even removed \vert and \Vert and replaced them with \rvert and \rVert.

I'd rather be able to tabout to > or | in the event they're used as closing brackets than not be able to tabout to them at all. Making the setting configurable does solve this issue though.